errors/next-head-count-missing.md

```diff
@@ -1,11 +0,0 @@
-# `next-head-count` is missing
```

Comment:
We need to keep this file and tell people there was an issue with 3rd party libs that was fixed in a newer version of Next.js and to please upgrade.